packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -254,6 +254,7 @@ class ThemeData with Diagnosticable {
     IconThemeData iconTheme,
     IconThemeData primaryIconTheme,
     IconThemeData accentIconTheme,
+    double iconButtonSplashRadius,
```

Comment:
I suspect that @HansMuller will suggest that this value should be on one of the subthemes, like IconTheme or ButtonTheme or some such, not on ThemeData itself. We're generally moving away from putting specific values on ThemeData because it got unwieldly.

Reply:
Ohh ok